schema/deploy/warehouse/site/data.sql

```diff
@@ -5,44 +5,113 @@ begin;
 
 insert into warehouse.site(identifier, details)
     values
+        ('Apple',                                   '{"category": "community",  "type": "apple-study", 
+            "sample_origin": "apple"}'),
+        ('AvalonFederalWayCareCenter',              '{"category": "community",  "type": "shelter", 
+            "swab_site": "avalon"}'),
         ('BirchTreeAcademy',                        '{"category": "community",  "type": "childcare"}'),
+        ('BoeingField',                             '{"category": "community",  "type": "shelter", 
+            "swab_site": "boeing"}'),
+        ('BurienNursingAndRehab',                   '{"category": "community",  "type": "nursingHome", 
```

Comment:
@truong97 I also saw this group of sites called "skilled nursing facility" or snf, instead of nursing home. Do you have a feel for which term is preferred/ more descriptive?

Reply:
@kschwabefry "skilled nursing facility" is more descriptive (I originally wasn't sure if it was too lengthy). I can change is to that if that is more preferable.